src/ConversationLearner.ts

```diff
@@ -74,7 +74,7 @@ export class ConversationLearner {
      * Provide an callback that will be invoked whenever a Session is started
      * @param target Callback of the form (context: BB.TurnContext, memoryManager: ClientMemoryManager) => Promise<void>
      */
-    public OnSessionStartCallback(target: OnSessionStartCallback) {
+    set OnSessionStartCallback(target: OnSessionStartCallback) {
```

Comment:
I'm not familiar with this syntax - what's the difference in usage between the former `public` and the new `set` variations ?

Reply:
Hmm, you can kin of ignore the `public` part. They're both public.
This changes it to be a setter to support assignment and also removes redundant keyword.

`public` keyword was a typescript syntax because IMO they were trying to appease developers coming from `C#` and needed contrast to `private`.  Both are not part of JS.

See the previous PR about EDC change for context